openncp-configuration-utility/Dockerfile

```diff
@@ -2,4 +2,7 @@ FROM eclipse-temurin:11-jre-jammy
 
 WORKDIR /usr/local/openncp-configuration-utility
 
+COPY openncp-configuration-utility.jar /usr/local/openncp-configuration-utility/openncp-configuration-utility.jar
+COPY openncp-configuration.properties /usr/local/openncp-configuration-utility/openncp-configuration.properties
```

Comment:
This should be created as a configmap, and created as a Projected Volume

https://kubernetes.io/docs/concepts/storage/projected-volumes/

Reply:
Fixed!